src/main/java/io/vertx/starter/config/util/ConfigUtil.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 Red Hat, Inc.
+ *
+ * Red Hat licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.vertx.starter.config.util;
+
+import io.vertx.core.json.Json;
+import io.vertx.core.json.JsonObject;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+
+/**
+ * @author Thomas Segismont
+ */
+public class ConfigUtil {
```

Comment:
Why you use an Util class to load a config. It should be a verticle input. I use to pull up as much as possible the configuration items.

Reply:
See the original issue description, I want to avoid having to manually update the configuration file on the starter machine.
I understand that external config allows to have different settings in production but in practice we don't want to do this, we want to deliver fixed, tested settings.